src/main/java/com/dev/cinema/Main.java

```diff
@@ -64,51 +68,54 @@ public static void main(String[] args) throws AuthenticationException {
-                .forEach(System.out::println);
+                .forEach(log::info);
         movieSessionService.findAvailableSessions(ironMan.getId(), LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(log::info);
         movieSessionService.findAvailableSessions(flash.getId(), LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(log::info);
 
         UserService userService = (UserService) injector.getInstance(UserService.class);
         User user1 = new User();
         user1.setEmail("arts@ukr.net");
         user1.setPassword("123");
         userService.add(user1);
-        System.out.println(userService.findByEmail(user1.getEmail()));
+        log.info("Find User by email " + userService.findByEmail(user1.getEmail()));
 
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         authenticationService.register("arts2@ukr.net", "1234");
-        System.out.println(authenticationService.login("arts2@ukr.net", "1234"));
+        log.info(authenticationService.login("arts2@ukr.net", "1234"));
 
         User user2 = userService.findByEmail("arts2@ukr.net").get();
 
         authenticationService.register("arts3@ukr.net", "12345");
-        System.out.println(authenticationService.login("arts3@ukr.net", "12345"));
+        log.info(authenticationService.login("arts3@ukr.net", "12345"));
 
         User user3 = userService.findByEmail("arts3@ukr.net").get();
 
         ShoppingCartService shoppingCartService =
                 (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
         ShoppingCart userShoppingCart3 = shoppingCartService.getByUser(user3);
         ShoppingCart userShoppingCart2 = shoppingCartService.getByUser(user2);
-        System.out.println(userShoppingCart3);
-        System.out.println(userShoppingCart2);
+        log.info("show Shopping cart by user # 3 " + userShoppingCart3);
+        log.info("show Shopping cart by user # 2 " + userShoppingCart2);
 
-        shoppingCartService.addSession(ironManSession,user3);
-        shoppingCartService.addSession(spiderManSession,user3);
-        System.out.println(shoppingCartService.getByUser(user3));
+        shoppingCartService.addSession(ironManSession, user3);
+        shoppingCartService.addSession(spiderManSession, user3);
+        log.info("show Shopping cart by user # 3 " + shoppingCartService.getByUser(user3));
 
-        shoppingCartService.addSession(flashSession,user2);
-        shoppingCartService.addSession(spiderManSession,user2);
-        System.out.println(shoppingCartService.getByUser(user2));
+        shoppingCartService.addSession(flashSession, user2);
+        shoppingCartService.addSession(spiderManSession, user2);
+        log.info("show Shopping cart by user # 3 " + shoppingCartService.getByUser(user2));
```

Comment:
a lot of cool "info" but never used "error" and "warn"

Reply:
agree. make sense